tests/publiccloud/download_repos.pm

```diff
@@ -36,11 +43,13 @@ sub run {
         my ($domain) = $parent    =~ '^([a-zA-Z.]*)';
         $ret = script_run "wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' --domains $domain --no-parent $parent $maintrepo", timeout => 600;
         if ($ret !~ /0|8/) {
-            die "wget error: The $maintrepo download failed with $ret return code.";
+            # softfailure, if repo doesn't exist (anymore). This is required for cloning jobs, because the original test repos could be empty already
```

Comment:
can't you just use `PUBLIC_CLOUD_SKIP_DOWNLOAD=1` in case of cloned job ? this change means that you won't get red flag anymore on quite severe problem . And experience showing that people tend to ignore softfailures ....

Reply:
The problem is a different one: When updates get released, their test repository becomes empty and a recent change in `zypper` broke out verification runs (See [poo#72121](https://progress.opensuse.org/issues/72121)).

This change is similar to the solution we had there - empty repositories don't break a verification run but remain visible as soft_failure.

Reply:
This is also a problem which should/does never occur with a properly scheduled run in OSD.
This problem affects cloned verification runs only